.editorconfig

```diff
@@ -23,14 +22,14 @@ space_after_anonymous_functions = false
 spaces_in_brackets = false
 
 # No Standard.  Please document a standard if different from .js
-[**.yml, **.html, **.css]
+[{*.yml,*.html,*.css}]
 trim_trailing_whitespace = true
 indent_style = tab
 
 # No standard.  Please document a standard if different from .js
-[**.md]
+[*.md]
 indent_style = tab
 
-# Standard at: 
+# Standard at:
 [Makefile]
```

Comment:
I removed the `Makefile` rule.  I didn't see any indentation in the `Dockerfile` or `Procfile`.  It looks `tab` indentation is meant to be used in all files.  `indent_style = tab` could be added to the `*` rule to apply to these files and all others if that's the case.